README.md

```diff
@@ -249,7 +249,57 @@ module.exports = {
     let checker = VersionChecker.forProject(this.project);
 
     for (let { name, root } = checker.allAddons()) {
-      // access to the add-on, in this case root + name
+      // access to the addon, in this case name and root
+    }
+  }
+};
+```
+### check
+
+A utility to verify that addons are installed at appropriate versions.  `npm` and `yarn resolve conflicting transitive dependency requirements by installing multiple versions.  They do not include a mechanism for packages to declare that a dependency must be unique.  This is, however, a practical constraint when building Ember applications (for example, we would not want to build an application that shipped two versions of Ember Data).
```

Comment:
> They do not include a mechanism for packages to declare that a dependency must be unique.

FWIW, https://github.com/npm/rfcs/pull/23 might be a reasonable thing to link to that would actually make things nicer here.